client/src/components/header/header.ts

```diff
@@ -0,0 +1,53 @@
+import styled, {keyframes} from "styled-components";
+import { Wrapper } from "../../box/flexBox";
+import {flipInX, swing} from 'react-animations'
+
+export const HeaderSection = styled.header`
+    background-color: rgba(0,0,0,0.65);
+    width: 100%;
+	height: 70px;
+    position: fixed;
+    padding-top: 20px;
+	z-index: 4;
+`
+export const HeaderWrapper = styled(Wrapper)`
+    justify-content: space-between;
+`
+
+export const LogoTitle = styled.div`
+    font-size: 30px;
+    line-height: 0.75;
+    color: #FFFFFF;
+`
+export const LogoSubtitle = styled.div`
+    font-size: 12px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+    letter-spacing: 0.680em;
+    line-height: 0.75;
+    margin-top: 12px;
+    padding-left: 5px;
+`
+
+export const Button = styled.button`
+    color: #FFFFFF;
+    padding: 10px 15px;
+    border-radius: 5px;
+    border: 2px solid #b49383;

```

Comment:
Probably it makes sense to create a special file, where you may define your design variables, for example:
```
export const themeConfig = {
    pallete: {
        primaryColor: 'some color',
        secondaryColor: 'another color HEX'
   },
  typography: {
       h1: {
            fontSize: ...
       },
       h2: {
            fontSize: ...
       }
  }
}
```
and use it in your styled component. If you decide to change color palette - it's easy to do via this file

Reply:
I will use a special file.
But this design i will use only in header.

Reply:
Common design patterns i will put on special files in  folders box and text

Reply:
I thinked about it: at first i didn't uderstand all sense of your order. Now I thihk it may have sense. I will do it.